File: android/app/src/main/java/org/mozilla/magnet/db/HistoryRecord.java
package org.mozilla.magnet.db;

import org.mozilla.magnet.db.History;

public class HistoryRecord {
    public int id;
    public String url;
    public long lastSeen;
    public long firstSeen;

    HistoryRecord(int id, String url, long lastSeen, long firstSeen) {
        this.id = id;
        this.url = url;
        this.lastSeen = lastSeen;
        this.firstSeen = firstSeen;
    }
}

File: ios/HistoryRecord.swift
//
//  HistoryRecord.swift
//  magnet
//
//  History struct to hold data for a record.
//
//  Created by Francisco Jordano on 18/07/2016.
//

import Foundation

struct HistoryRecord {
  var id: Int64
  var url: String
  var lastSeen: Int64
  var firstSeen: Int64
}